clipctl

```diff
@@ -1,27 +1,64 @@
 #!/usr/bin/env bash
 
+: "${CM_DIR:="${XDG_RUNTIME_DIR-"${TMPDIR-/tmp}"}"}"
+
 if [[ -z $1 ]] || [[ $1 == --help ]] || [[ $1 == -h ]]; then
     cat << 'EOF'
 clipctl provides controls for the clipmenud daemon.
 
 You can temporarily disable clip collection without stopping clipmenud entirely
 by running "clipctl disable". You can then reenable with "clipctl enable".
+
+Commands:
+    enable: enable clip collection
+    disable: disable clip collection
+    status: returns "enabled" or "disabled"
+    toggle: toggles clip collection
+    version: returns major version
+    cm-dir: returns the directory used for caching
```

Comment:
Oh, I like this and don't know why it never occurred to me that this might be a good place to centralise. Thanks!

Reply:
Only mild comment, I think I prefer "cache-dir" since it's what we call it everywhere else.

Reply:
Changed to `cache-dir`